### PR TITLE
Update grammar in advanced-solidus/state-machines

### PR DESCRIPTION
**Pull request by lsizani**
## Summary

- Added missing "O" in "or"
- Removed "for" in "look for through ..."
- Updated Line number to where the OrderMerger class config option is on app_configuration

## Checklist

- [ ] I have followed the [Diátaxis](https://diataxis.fr/) framework in my PR.
- [x] I have verified that the preview environment works correctly.

### CODE REVIEW

**Review by elia — APPROVED** [review]
Awesome!

File: README.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ $ yarn start
...
 
 This command starts a local development server and opens up a browser window. Most changes are reflected live without
-having to restart the server.
+having to restart the server. This guide is versioned, so your live changes will be on http://localhost:3000/next.
 
 ## Versions
 
 The documentation in this repo is versioned for every major and minor release.
-All changes merged to the main docs (under `docs/`) will refer to the `master` branch of the Solidus repo.
+All changes merged to the main docs (under `docs/`) will refer to the `main` branch of the Solidus repo.
```
[the remark this answers is not in the document]
Comment (elia) [reply README.md:27]: oh, that's right, the original text was correct here

**Review by kennyadsl — APPROVED** [review]
Thanks!

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lsizani) [reply README.md]: I assumed this sentence was about this guide 🤦🏽

**Pull request merged**